homeassistant/components/incomfort/quality_scale.yaml

```diff
@@ -0,0 +1,69 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
```

Comment:
I think we now also have a trend where we remove the domain name (`incomfort`) and just replace it with the full name, as the user doesn't encounter the domain name as much as they did back in the YAML setup days

Reply:
Removed the term.